### PR TITLE
Add PopHub.app

### PR DESCRIPTION
**Pull request by sachin21**
### Changes to a cask

Added PopHub.
PopHub is a GitHub activity & notification viewer.

GitHub: https://github.com/questbeat/PopHub-Electron 
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/pophub.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'pophub' do
+  version '0.0.1'
+  sha256 '5471dee854bed85394381a28ab60875c4d0196cf69b41cf63a8ce4a61cbff7c0'
+
+  url 'https://github.com/questbeat/PopHub-Electron/releases/download/v#{version}/PopHub-darwin-x64.zip'
```
Comment (jawshooah) [inline Casks/pophub.rb]: You're using interpolation here, so you need to use double quotes.

Comment (adidalal) [inline Casks/pophub.rb]: Also, mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

Comment (sachin21) [inline Casks/pophub.rb]: Thank you for the review! I fixed it. Could you re-check this?

**Comment by sachin21** [comment]
😄 Thanks for quickly merging this pull request.

**Pull request merged**